src/main/java/com/checkout/workflows/four/actions/response/WorkflowActionType.java

```diff
@@ -0,0 +1,13 @@
+package com.checkout.workflows.four.actions.response;
```

Comment:
I think the right place for this enum is under `com.checkout.workflows.four.actions`, along with `WorkflowActionType`, wdyt?